src/externals/scheduler.js

```diff
@@ -1,9 +1,20 @@
 const axios = require('axios').default
 
+const { GATEWAY_ADDRESS } = process.env
+
 module.exports = class Scheduler {
   constructor() {
     this.axios = axios.create({
-      timeout: 1000
+      timeout: 1000,
+      validateStatus: null,
     })
   }
+
+  async createJob(interval, payload) {
```

Comment:
```suggestion
  async createWatchJob(interval, payload) {
```
Viet tuong minh ra, vi sau nay scheduler service se co them nhieu loai job khac nua day